operations/security_and_authentication/Change_Air-Cooled_Node_BMC_Credentials.md

```diff
@@ -1,50 +1,31 @@
 # Change Air-Cooled Node BMC Credentials
 
-This procedure will use the System Configuration Service (SCSD) to change all air-cooled Node BMCs in the system to the same global credential.
+This procedure describes how to use the System Admin Toolkit's (SAT) `sat bmccreds`
+command to set a global credential for all BMCs on air-cooled nodes.
 
 ### Limitations
 
 All air-cooled and liquid-cooled BMCs share the same global credentials. The air-cooled Slingshot switch controllers (Router BMCs) must have the same credentials as the liquid-cooled Slingshot switch controllers.
 
+The `sat bmccreds` command is not able to target air-cooled nodes directly. It can, however, target nodes by their component name (xname). The following procedure uses xnames to target nodes.
+
 ### Prerequisites
 
--   The Cray command line interface \(CLI\) tool is initialized and configured on the system.
--   Review procedures for SCSD in the [Cray System Management (CSM) Administration Guide](../index.md#system-configuration-service)
+SAT is installed and configured.
 
 ### Procedure
 
-1.  Set the `NEW_BMC_CREDENTIAL` to specify the new root user password for air-cooled node BMCs:
-    ```bash
-    ncn-m001# read -s NEW_BMC_CREDENTIAL
-    ncn-m001# echo $NEW_BMC_CREDENTIAL
-    ```
+1. Get the xnames for all air-cooled nodes.
 
-    Expected output:
-    ```
-    new.root.password
-    ```
+    The following operation will store the xnames in a variable named `RIVER_NODEBMC_XNAMES`.
 
-2.  Create an SCSD payload file to change all air-cooled node BMCs to the same global credential:
     ```bash
-    ncn-m001# cat > bmc_creds_glb.json <<DATA
-    {
-        "Force":false,
-        "Username": "root",
-        "Password": "$NEW_BMC_CREDENTIAL",
-        "Targets":
-        $(cray hsm state components list --class River --type NodeBMC --format json | jq -r '[.Components[] | .ID]')
-    }
-    DATA
+    RIVER_NODEBMC_XNAMES=$(cray hsm state components list --class River --type NodeBMC \
+        --format json | jq -r '[.Components[] | .ID ]| join(",")')
     ```
 
-    Inspect the generated SCSD payload file:
-    ```bash
-    ncn-m001# cat bmc_creds_glb.json | jq
-    ```
+2. Set the same random password for every BMC on an air-cooled node.
```

Comment:
I guess this is just consistent with how the numbering was before, but you can just do ones for everything, e.g.:
```
1. first ordered list item
1. second ordered list item
...
```
And github will automatically number them.